### PR TITLE
Editors should not be able to manage feed collaborators.

### PR DESCRIPTION
**Pull request by caiosba**
## Description

I had to wrap the `<SaveFeed />` component in a new `<CreateFeed />` component, which is basically a query renderer, and that now responds to the `feed/create` route. This way I'm able to get the permissions.

The rest of the code is basically checking for permissions and then rendering components or not based on them.

Reference: CV2-3953.

## Type of change

- [ ] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

Manually.

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [x] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [ ] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

File: src/app/components/feed/CreateFeed.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    query={graphql`
+      query CreateFeedQuery {
+        team {
+          permissions
+        }
+      }
+    `}
+    render={({ error, props }) => {
+      if (!error && props) {
+        return (<SaveFeed permissions={JSON.parse(props.team.permissions)} />);
```
Comment (amoedoamorim) [inline src/app/components/feed/CreateFeed.js:18]: Maybe `safelyParseJSON` instead?

Comment (caiosba) [reply src/app/components/feed/CreateFeed.js]: Absolutely! Done :)

File: src/app/components/feed/FeedCollaboration.js [hunk 2]
```diff
@@ -276,6 +278,18 @@ const FeedCollaboration = ({
...
+          variant="warning"
+          title={
+            <FormattedMessage
+              id="feedCollaboration.editorWarningCreateFeed"
+              defaultMessage="Contact your workspace admin to add other organizations to this shared feed."
+              description="Warning displayed on feed collaboration box on create feed page when user is an editor and can't invite people to the feed."
+            />
+          }
+        />
+      )}
```
Comment (amoedoamorim) [inline src/app/components/feed/FeedCollaboration.js:292]: Does this mean someone without permission to invite others could be able to create a feed anyway?

Comment (caiosba) [reply src/app/components/feed/FeedCollaboration.js:292]: Yes :shrug:

Comment (brianfleming) [reply src/app/components/feed/FeedCollaboration.js:292]: Yes @amoedoamorim we don't want to stop editors from being able to push work forward, but things get a bit more sensitive when it comes to outside contributors, so we want admins to handle that

**Review by amoedoamorim — APPROVED** [review]

**Pull request merged**